<!--REQUEST_ID=1d9b09a7-792f-4fe0-b150-827189e15712-->
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.49690&attempt_id=b6eb4205-8a31-42df-9d24-cfc13d4a2829)

Attempted: firefox-esr-60

<details><summary>Partial log (click to expand)</summary><p>

```
 0:07.26     errread, errwrite)
 0:07.26   File "/nix/store/c6mnc84yva1wa85nzjfha6mxfsk6qbg7-python-2.7.15/lib/python2.7/subprocess.py", line 1047, in _execute_child
 0:07.26     raise child_exception
 0:07.26 OSError: [Errno 2] No such file or directory
 0:07.29 *** Fix above errors and then restart with\
 0:07.29                "/nix/store/f5aw0jl41r3nydav2mprvd2zyb76hg92-gnumake-4.2.1/bin/make -f client.mk build"
 0:07.29 make: *** [client.mk:149: configure] Error 1
builder for '/nix/store/xy2q351kpvrvrb62736bm81754fi6v74-firefox-esr-unwrapped-60.3.0esr.drv' failed with exit code 2
cannot build derivation '/nix/store/x5dw982sw8mgfkwdid9if2cbc6j8hgwm-Firefox-60.3.0esr.drv': 1 dependencies couldn't be built
error: build of '/nix/store/x5dw982sw8mgfkwdid9if2cbc6j8hgwm-Firefox-60.3.0esr.drv' failed
```
</p></details>

--- a/pkgs/applications/networking/browsers/firefox/common.nix
+++ b/pkgs/applications/networking/browsers/firefox/common.nix
@@ -196,8 +196,7 @@ stdenv.mkDerivation rec {
   ]
   ++ lib.optional (stdenv.isDarwin && lib.versionAtLeast ffversion "61") "--disable-xcode-checks"
   ++ lib.optional (lib.versionOlder ffversion "61") "--enable-system-hunspell"
-  ++ lib.optionals (lib.versionAtLeast ffversion "56" && !stdenv.hostPlatform.isi686) [
-    # on i686-linux: --with-libclang-path is not available in this configuration
+  ++ lib.optionals (lib.versionAtLeast ffversion "56") [
     "--with-libclang-path=${llvmPackages.libclang}/lib"
     "--with-clang-path=${llvmPackages.clang}/bin/clang"
   ]

--- a/pkgs/applications/networking/browsers/firefox/packages.nix
+++ b/pkgs/applications/networking/browsers/firefox/packages.nix
@@ -8,12 +8,6 @@ let
     ./env_var_for_system_dir.patch
   ];
 
-  firefox60_aarch64_skia_patch = fetchpatch {
-    name = "aarch64-skia.patch";
-    url = https://src.fedoraproject.org/rpms/firefox/raw/8cff86d95da3190272d1beddd45b41de3148f8ef/f/build-aarch64-skia.patch;
-    sha256 = "11acb0ms4jrswp7268nm2p8g8l4lv8zc666a5bqjbb09x9k6b78k";
-  };
-
 in
 
 rec {
@@ -82,7 +76,7 @@ rec {
       # this one is actually an omnipresent bug
       # https://bugzilla.mozilla.org/show_bug.cgi?id=1444519
       ./fix-pa-context-connect-retval.patch
-    ] ++ lib.optional stdenv.isAarch64 firefox60_aarch64_skia_patch;
+    ];
 
     meta = firefox.meta // {
       description = "A web browser built from Firefox Extended Support Release source tree";

